packages/backend/src/services/quadlet-service.ts

```diff
@@ -461,7 +460,7 @@ export class QuadletService extends QuadletHelper implements Disposable, AsyncIn
     if (quadlet.type !== QuadletType.KUBE)
       throw new Error(`cannot get kube yaml of non-kube quadlet: quadlet ${quadlet.id} type is ${quadlet.type}`);
 
-    if (!quadlet.content || !quadlet.service)
+    if (!quadlet.service)
```

Comment:
this change is not related to this PR, you may use `isServiceQuadlet` but otherwise this has to be reverted